lib/gdbstub.c

```diff
@@ -276,9 +276,22 @@ void process_xfer(gdbstub_t *gdbstub, char *s)
     printf("xfer = %s %s\n", name, args);
 #endif
     if (!strcmp(name, "features") && gdbstub->arch.target_desc != NULL) {
-        /* FIXME: We should check the args */
+        /* check the args */
+        char *action =  strtok(args, ":");
+        assert(strcmp(action, "read") == 0);
+        char *filename = strtok(NULL, ":");
```

Comment:
Change the variable name to `annex` to match [qXfer-target-description-read](https://sourceware.org/gdb/current/onlinedocs/gdb.html/General-Query-Packets.html#qXfer-target-description-read)